Code/GraphMol/MonomerMol/MonomerMol.cpp

```diff
@@ -0,0 +1,339 @@
+/* -------------------------------------------------------------------------
+ * Declares schrodinger::rdkit_extensions:: tools for Coarse-grain ROMols
+ *
+ * A "coarse grain" (CG) ROMol uses RDKit atoms to represent monomers. Chains
+ * are represented by COP Substance Groups on the ROMol.
+ *
+ * For use with functionality in schrodinger::rdkit_extensions
+ *
+ *
+ * Copyright Schrodinger LLC, All Rights Reserved.
+ --------------------------------------------------------------------------- */
+
+#include "MonomerMol.h"
+
+#include <GraphMol/QueryAtom.h>
+#include <GraphMol/ROMol.h>
+#include <GraphMol/RWMol.h>
+#include <GraphMol/SubstanceGroup.h>
+#include <GraphMol/MonomerInfo.h>
+
+#include <boost/functional/hash.hpp>
+
+template <typename T>
+static bool is_dummy_obj(const T& obj)
```

Comment:
saw a compiler error about this. probably remove `obj` or mark it as `[[maybe_unused]]`